### ISSUE TITLE
This network environment has no discovery queue configured.

### ISSUE THREAD

**Issue by steve-netskope** [allegro/ralph#1395]
what is the way to add a discovery queue on docker instance to scan network?

"This network environment has no discovery queue configured."

![screen shot 2015-05-04 at 4 57 22 pm](https://cloud.githubusercontent.com/assets/7399765/7465056/cdf0ad82-f27e-11e4-9d86-1c45e339a59e.png)

**Comment by steve-netskope** [comment]
got it fixed.

Quesion: DC and Rack name is different from 'Racks' and 'DC View' tabs. Why is that and how are they updated from admin section?

**Comment by vi4m** [comment]
Discovery -> Device type Rack / DC
Assets -> model Rack / DC

the're related in some way, because assets & discovery are separate apps.

**Comment by vi4m** [comment]
Cutting edge version(Ralph-NG) does not require to create separate objects, all apps are integrated into one model. We're aware of this inconvienece.

**Comment by steve-netskope** [comment]
Where can I get Ralph-NG?

**Comment by vi4m** [comment]
We will try to publish it next week. Here are some details https://github.com/allegro/ralph/wiki/Ralph-3.0-development

**Comment by steve-netskope** [comment]
Thank you!

**Comment by steve-netskope** [comment]
Would there be timeline when Ralph NG may be in beta state with docker image ready?

**Comment by vi4m** [comment]
The second milestone is: dc visulization and manual inventory management(assets) - without scan.
Here's work progress: 

https://waffle.io/allegro/ralph?milestone=2.%20Visualisation%20%2B%20tabs%20mechanism%20research

**Comment by vi4m** [comment]
We're building docker images daily - you can check current build on the demo site
http://ralph.mkurek.com  
login: ralph password: ralph

Features: 
- full back office and dc manual asset tracking
- dc visualization
- permissions and fields limiting
- exporting and importing from Ralph 2.x

If you want to help us testing, I prepared deb packages for ubuntu 14.04 - currently documenting it, will update in this thread.

In short, to install it from deb package just issue:

```
sudo apt-key adv --keyserver  hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
sudo echo "deb https://dl.bintray.com/vi4m/ralph wheezy main" >  /etc/apt/sources.list.d/vi4m2_ralph.list
```

It will install ralph into /opt location.

**Comment by vi4m** [comment]
Disclaimer: we need to clean up login page, home screen, and many touches around.

**Comment by mkurek** [comment]
Problem solved.
